### PR TITLE
Update clear method to clear in navbar

### PR DESCRIPTION
**Pull request by mnouwens**
Update clear method to clear the div and not the whole panel. This keeps the structure intact.

### CODE REVIEW

File: gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java [hunk 1]
```diff
@@ -83,8 +90,17 @@ public MaterialNavBar() {
         navMenu.setTextAlign(TextAlign.CENTER);
         navMenu.setIconPosition(IconPosition.NONE);
     }
+    
+    
 
     @Override
+	public void clear() {
+    	div.clear();
```
Comment (BenDol) [inline gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java]: Tabulation

File: gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java [hunk 2]
```diff
@@ -83,8 +90,17 @@ public MaterialNavBar() {
...
+	}
+
+
+
```
Comment (BenDol) [inline gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java]: Remove the new line

**Comment by mnouwens** [comment]
Update done. The pull request get's updated automaticlly?

**Comment by kevzlou7979** [comment]
Yeah

**Comment by mnouwens** [comment]
Ok Getting the hang of this. :-)

File: gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java [hunk 3]
```diff
@@ -85,6 +92,11 @@ public MaterialNavBar() {
     }
 
     @Override
+	public void clear() {
+    	div.clear();
```
Comment (BenDol) [inline gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java]: Need to tabulate this line

Comment (mnouwens) [inline gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java]: You mean all spaces or 2 tabs. I saw that when creating eclipse users spaces and i then used tab.

Whais should it be?

Comment (BenDol) [inline gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java]: Should look like this (with spaces):
```java
public void clear() {
        div.clear();
```

Comment (BenDol) [inline gwt-material/src/main/java/gwt/material/design/client/ui/MaterialNavBar.java]: So you need to change your tabulation from tabs to spaces (4 spaces per tab)

**Comment by mnouwens** [comment]
Updates all done?

**Comment by BenDol** [comment]
1 tab = 4 spaces so in this code there should be 8 spaces to tabulate it properly.

**Comment by kevzlou7979** [comment]
Will close this and create anew PR , We are going to release 1.5.1 now

**Comment by kevzlou7979** [comment]
AdFixed via https://github.com/GwtMaterialDesign/gwt-material/commit/53833e78415c465cd179b109671bcf50774dc8ce

**Pull request closed without merging**